BLUESPAWN-common/headers/common/wrappers.hpp

```diff
@@ -35,7 +40,7 @@ class GenericWrapper {
 
 public:
 
-	GenericWrapper(T object, void(*freeFunction)(T) = [](T object){ delete object; }, T BadValue = nullptr)
+	GenericWrapper(T object, std::function<void(T)> freeFunction = [](T object){ delete object; }, T BadValue = nullptr)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'GenericWrapper < HANDLE >' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4967854)